packages/0xsequence/tests/browser/mock-wallet/mock-wallet.test.ts

```diff
@@ -42,15 +42,15 @@ const main = async () => {
     {
       name: 'hardhat',
       chainId: 31337,
-      rpcUrl: provider.connection.url,
+      rpcUrl: provider._getConnection().url,
```

Comment:
this is kinda weird.. we shouldn't need to use `_` methods.. is there nothing else with this info on the object..?